Makefile

```diff
@@ -161,7 +161,8 @@ install-sysvinit: install-init
 	install network/qubes-iptables $(DESTDIR)/etc/init.d/
 
 install-rh: install-systemd install-systemd-dropins install-sysvinit
-	install -D -m 0644 misc/qubes-r4.repo $(DESTDIR)/etc/yum.repos.d/qubes-r4.repo
+	install -D -m 0644 misc/qubes-r4.repo.in $(DESTDIR)/etc/yum.repos.d/qubes-r4.repo
+	sed -i "s/@DIST@/$(shell echo $(DIST) | sed 's/[[:digit:]]//g')/g" $(DESTDIR)/etc/yum.repos.d/qubes-r4.repo
```

Comment:
what about `${DIST%%[0-9]*}` instead of `$(shell ...)`?

Reply:
Oh I didn't know such syntax in Makefile, in Bash I would have used ${DIST//[[:digit:]]}. So I tried but got empty result. Will check.

Reply:
Yes, I meant shell syntax for that. You may need to use `$$` instead of `$`.